plugins/database/src/db_plugin.cpp

```diff
@@ -13912,28 +13913,49 @@ namespace
 
         // Modify ticket operation.
         if (std::strcmp(_op_name, "mod") == 0) {
+            int i{0};
+            char myTime[TIME_LEN]{}; // NOLINT(cppcoreguidelines-avoid-c-arrays, modernize-avoid-c-arrays)
+
             if (std::strcmp(_arg3, "uses") == 0) {
-                cllBindVars[0] = _arg4;
-                cllBindVars[1] = ticket_id_string.c_str();
-                cllBindVarCount = 2;
+                getNowStr(myTime); // NOLINT(cppcoreguidelines-pro-bounds-array-to-pointer-decay)
+                i = 0;
+                // NOLINTNEXTLINE(cppcoreguidelines-pro-bounds-constant-array-index)
+                cllBindVars[i++] = _arg4;
+                // NOLINTNEXTLINE(cppcoreguidelines-pro-bounds-constant-array-index)
+                cllBindVars[i++] = myTime; // NOLINT(cppcoreguidelines-pro-bounds-array-to-pointer-decay)
```

Comment:
Clang-format would complain because it would go over the line length check. Also correct me if I am wrong but in 851cfc98277ad5968674d7a05bcf262fac666fa6 `
 -cppcoreguidelines-pro-bounds-array-to-pointer-decay` was not demoted to a warning as it is still in the "Warnings As Errors" section.

Reply:
That linter option was demoted.

The leading minus sign means **remove**.

Reply:
I rebased to include the changes from https://github.com/irods/irods/commit/851cfc98277ad5968674d7a05bcf262fac666fa6 yet I still see `cppcoreguidelines-pro-bounds-array-to-pointer-decay` errors in clang-tidy.